website/blog/2019-08-30-thorium.md

```diff
@@ -0,0 +1,148 @@
+---
+author: Tomasz Godzik
+title: Metals v0.7.1 - Thorium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are excited to announce the release of Metals v0.7.0 - codename "Thorium" 🎉
+The release includes mostly bug fixes with some smaller new features.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+In this release we merged 39 PRs and closed 11 issues, full details:
+https://github.com/scalameta/metals/milestone/12?closed=1
+
+## TL;DR
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+- support for Scala 2.12.9
+- fix completions in case of inline comments
+- add support for non-directory sources
+- automatically add package definition to new source files
+- automatically insert pipes in multiline string
+- a lot of miscellaneous fixes
+
+## Fix completions in case of inline comments
+
+Previously, completions after a comment and newline would yield no results:
+
+```scala
+val List(1,2,3)
+  .map(num => num + 1) // we add one
```

Comment:
let's keep the ammount of noise to minimum. I read the comment as a part of the description of this feature and got confused for a moment :(

```suggestion
  .map(num => num + 1) // some comment
```